src/Assignment-1/lib/math.js

```diff
@@ -0,0 +1,19 @@
+const math = require("lodash");
+
+const add = (num1, num2) => {
+  return math.add(num1, num2);
+};
+
+const sub = (num1, num2) => {
+  return math.subtract(num1, num2);
+};
+
+const mult = (num1, num2) => {
+  return math.multiply(num1, num2);
+};
+
+const div = (num1, num2) => {
+  return math.divide(num1, num2);
+};
+
```

Comment:
move lib and utils on root folder -for e.g. src/lib

Reply:
done